gaphor/settings.py

```diff
@@ -0,0 +1,34 @@
+"""Application settings support for Gaphor."""
+
+from gi.repository import Gio
+import logging
+
+APPLICATION_ID = "org.gaphor.Gaphor"
```

Comment:
This variable is also defined in `gaphor/ui/__init__.py`. There we should import this value instead.

Reply:
Hi @amolenaar, this creates a circular import that's why I redefined it:

gaphor.ui -> gaphor.application -> gaphor.core -> gaphor.i18n -> gaphor.settings -> gaphor.ui

Reply:
And if you define `APPLICATION_ID` here and import it in `gaphor.ui`?

Reply:
I swapped this, good idea!